### PR TITLE
React/react1/week1

### PR DESCRIPTION
**Pull request by Shweta-MG**

### CODE REVIEW

File: react/react1/week1/todo-list/src/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Navbar from './Navbar';
+import Home from './Home';
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
```
Comment (LucasIversen) [inline react/react1/week1/todo-list/src/App.js:7]: Cool to see you try out a navbar 👍

File: react/react1/week1/todo-list/src/Home.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import ToDoList from "./ToDoList";
+
+
+const Home = () => {
+    const [todos, setTodos] = useState([
```
Comment (LucasIversen) [inline react/react1/week1/todo-list/src/Home.js:6]: good use of state

File: react/react1/week1/todo-list/src/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.navbar {
```
Comment (LucasIversen) [inline react/react1/week1/todo-list/src/Navbar.css:1]: Good to see you try out some styling in react also 🥇

File: react/react1/week1/todo-list/src/ToDoList.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import './ToDoList.css';
+
+const ToDoList = (props) => {
+    const todos = props.todos;
+    const title = props.title;
+    
+    return (  
+        <div className="todo-list">
+            <h2>{ title }</h2>
+            {todos.map((todo) => (
```
Comment (LucasIversen) [inline react/react1/week1/todo-list/src/ToDoList.js:10]: good use of map 👍

**Review by LucasIversen — APPROVED** [review]
Really good job 👍 keep it up 💪

**Comment by Shweta-MG** [comment]
@LucasIversen Thank you very much for your time and encouraging comments.

**Pull request merged**